public/bundle.js

```diff
@@ -0,0 +1,540 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
It's better to leave bundle files uncommited

Reply:
added bundles to git ignore